server/characters/characters_common.py

```diff
@@ -31,7 +31,7 @@ def db_connection():
     db_pass = "allDatSQL"
     if config.has_option('Characters','characters_psql_pass'):
         db_pass = config.get('Characters','characters_psql_pass')
-    connection = psycopg2.connect("dbname=%s user=%s password=%s" % (db, username,db_pass))
+    connection = psycopg2.connect("dbname=%s user=%s password=%s host=localhost" % (db, username, db_pass))
```

Comment:
We shouldn't hard-code this. I'll add an issue to make that configurable.

Reply:
I agree. I forgot about it before I made the commit.